api/types/backend/build.go

```diff
@@ -7,6 +7,15 @@ import (
 	"github.com/docker/docker/pkg/streamformatter"
 )
 
+const (
+	// PullFlagNoPull only returns local images
+	PullFlagNoPull int = iota
```

Comment:
Maybe a `type PullOption int` here instead of int so the supported values are easier to discover?